EstateAdministrationUI.IntegrationTests/Common/TestingContext.cs

```diff
@@ -0,0 +1,87 @@
+﻿namespace EstateAdministrationUI.IntegrationTests.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using SecurityService.DataTransferObjects.Responses;
+    using Shared.IntegrationTesting;
+    using Shared.Logger;
+    using Shouldly;
+    using TechTalk.SpecFlow;
+
+    public class TestingContext
+    {
+        public EstateDetails GetEstateDetails(TableRow tableRow, Guid? testId = null)
+        {
+            String estateName = SpecflowTableHelper.GetStringRowValue(tableRow, "EstateName").Replace("[id]", testId.Value.ToString("N"));
+
+            EstateDetails estateDetails = this.Estates.SingleOrDefault(e => e.EstateName == estateName);
+
+            estateDetails.ShouldNotBeNull();
+
+            return estateDetails;
+        }
+
+        public List<Guid> GetAllEstateIds()
+        {
+            return this.Estates.Select(e => e.EstateId).ToList();
+        }
+
+        public EstateDetails GetEstateDetails(String estateName)
+        {
+            EstateDetails estateDetails = this.Estates.SingleOrDefault(e => e.EstateName == estateName);
+
+            estateDetails.ShouldNotBeNull();
+
+            return estateDetails;
+        }
+
+        private List<EstateDetails> Estates;
+        public void AddEstateDetails(Guid estateId, String estateName)
+        {
+            this.Estates.Add(EstateDetails.Create(estateId, estateName));
+        }
+
+        public String AccessToken { get; set; }
+
+        public DockerHelper DockerHelper { get; set; }
+
+        public NlogLogger Logger { get; set; }
+
+        public Dictionary<String, Guid> Users;
+        public Dictionary<String, Guid> Roles;
+
+        public List<ClientDetails> Clients;
+
+        public List<String> ApiResources;
+        public List<String> IdentityResources;
+
+        public TokenResponse TokenResponse;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)